lib/client/keyagent_test.go

```diff
@@ -114,7 +114,7 @@ func (s *KeyAgentTestSuite) TestAddKey(c *check.C) {
 		s.username,                            // private key
 		s.username + constants.FileExtPub,     // public key
 		s.username + constants.FileExtTLSCert, // Teleport TLS certificate
-		filepath.Join(s.username+sshDirSuffix, s.key.ClusterName+constants.FileExtSSHCert), // SSH certificate
+		filepath.Join(s.username+constants.SSHDirSuffix, s.key.ClusterName+constants.FileExtSSHCert), // SSH certificate
```

Comment:
@Joerger Any way to move the proper path calc login into a helper function and use it everywhere instead? You had to change it in 4 places. Would help prevent situations like this in future.

Reply:
It's hard to make helper functions for these file name constants that covers all cases because we use them differently across our code base. I'll make helper functions specific to the profile loading which this PR covers, but this won't avoid issues since the read/write logic is totally separate right now. Opened issue for this - #6233